packages/office-ui-fabric-react/src/components/DetailsList/DetailsHeader.tsx

```diff
@@ -150,38 +150,48 @@ export class DetailsHeader extends BaseComponent<IDetailsHeaderProps, IDetailsHe
         direction={ FocusZoneDirection.horizontal }
       >
         { showCheckbox ? (
-          <div
-            className={ css(
-              'ms-DetailsHeader-cell',
-              'ms-DetailsHeader-cellIsCheck',
-              styles.cell,
-              styles.cellIsCheck,
-              checkStyles.owner,
-              isAllSelected && checkStyles.isSelected
-            ) }
-            onClick={ this._onSelectAllClicked }
-            aria-colindex={ 0 }
-            role='columnheader' >
-            <span
-              aria-label={ ariaLabelForSelectionColumn }
-            ></span>
-            {
-              onRenderColumnHeaderTooltip({
-                hostClassName: css(styles.checkTooltip),
-                id: `${this._id}-checkTooltip`,
-                setAriaDescribedBy: false,
-                content: ariaLabelForSelectAllCheckbox,
-                children: (
-                  <DetailsRowCheck
-                    aria-describedby={ `${this._id}-checkTooltip` }
-                    selected={ isAllSelected }
-                    anySelected={ false }
-                    canSelect={ true }
-                  />
-                )
-              }, this._onRenderColumnHeaderTooltip)
-            }
-          </div >
+          [

```

Comment:
I just added tslint-react which has a rule that can catch this. I have it disabled right now, but I'm going to have someone on our team work on removing lint rule suppressions.